webpack/webpack.prod.js

```diff
@@ -0,0 +1,17 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common');
+
+module.exports = merge(common, {
+  mode: 'production',
+
+  output: {
+    filename: '[name].[chunkhash].js',
+    chunkFilename: '[name].[chunkhash].chunk.js',
+    publicPath: '/',
+  },
+
```

Comment:
Should we add some plugins for optimization in this stage? 
https://webpack.js.org/guides/production/

Reply:
It's for the webpack-production story